src/MainWindow.vala

```diff
@@ -51,7 +50,7 @@ public class Tasks.MainWindow : Gtk.ApplicationWindow {
         add_action_entries (ACTION_ENTRIES, this);
 
         foreach (var action in action_accelerators.get_keys ()) {
-            ((Gtk.Application) GLib.Application.get_default ()).set_accels_for_action (ACTION_PREFIX + action, action_accelerators[action].to_array ());
+            ((Gtk.Application) GLib.Application.get_default ()).set_accels_for_action (ACTION_PREFIX + action, action_accelerators[action].to_array ());  // vala-lint=line-length
```

Comment:
I'd prefer not to add exclusions for line length. We can probably do something about this to make it not so long

Reply:
Using two lines now for this.